dotNETStandard/HealthVaultProviderManagementPortal/HealthVaultProviderManagementPortal/Controllers/PatientController.cs

```diff
@@ -194,5 +251,13 @@ private static JsonSerializerSettings GetDeserializationSettings()
 
             return settings;
         }
+
+        private static JsonSerializerSettings GetSerializationSettings()
```

Comment:
Since there's only a single setting made, I would prefer:

`{
    return new JsonSerializerSettings().ConfigureForNodaTime(DateTimeZoneProviders.Tzdb);
}`